airbyte-integrations/connectors/source-uservoice/manifest.yaml

```diff
@@ -0,0 +1,4174 @@
+          type: RequestOption
+          field_name: updated_after
+          inject_into: request_parameter
+        cursor_datetime_formats:
+          - "%Y-%m-%dT%H:%M:%SZ"
+    labels:
+      type: DeclarativeStream
+      name: labels
+      retriever:
+        type: SimpleRetriever
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            field_name: cursor
+            inject_into: request_parameter
+          pagination_strategy:
+            type: CursorPagination
+            cursor_value: "{{ response.get('pagination', {}).get('cursor') }}"
+            stop_condition: "{{ response.get('pagination', {}).get('cursor') is not defined }}"
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /admin/labels
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - labels
+      primary_key:
+        - id
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/labels"
+      incremental_sync:
+        type: DatetimeBasedCursor
+        cursor_field: updated_at
+        end_datetime:
+          type: MinMaxDatetime
+          datetime: "{{ now_utc().strftime('%Y-%m-%dT%H:%M:%SZ') }}"
+          datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+        start_datetime:
+          type: MinMaxDatetime
+          datetime: "{{ config[\"start_date\"] }}"
+          datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+        datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+        end_time_option:
+          type: RequestOption
+          field_name: updated_before
+          inject_into: request_parameter
+        start_time_option:
+          type: RequestOption
+          field_name: updated_after
+          inject_into: request_parameter
+        cursor_datetime_formats:
+          - "%Y-%m-%dT%H:%M:%SZ"
+    comments:
+      type: DeclarativeStream
+      name: comments
+      retriever:
+        type: SimpleRetriever
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            field_name: cursor
+            inject_into: request_parameter
+          pagination_strategy:
+            type: CursorPagination
+            cursor_value: "{{ response.get('pagination', {}).get('cursor') }}"
+            stop_condition: "{{ response.get('pagination', {}).get('cursor') is not defined }}"
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /admin/comments
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - comments
+      primary_key:
+        - id
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/comments"
+      incremental_sync:
```

Comment:
The default sorting method the API does is using the `id` you must change that to use  `updated_at` to make incremental to work properly.
<img width="1412" alt="image" src="https://github.com/user-attachments/assets/9be6b79c-9aed-43d5-a0d5-fea90f397fe7">

Reply:
added, but few streams don't support `updated_at` as value for sort param